src/main.rs

```diff
@@ -69,12 +69,12 @@ pub struct Options<'a> {
     pub paging: bool,
 }
 
-enum OutputType<'a> {
+enum OutputType {
     Pager(Child),
-    Stdout(StdoutLock<'a>),
+    Stdout(io::Stdout),
```

Comment:
cool! I always wanted to do this. Will not locking the output cause any issues?

Reply:
In my understanding, this would only cause problems if we wanted to write to stdout from different threads. Also, keeping a lock might be faster... but we are limited by the syntax highlighting speed anyway, so I believe this can be ignored.